docs/guides/go-built-in.md

```diff
@@ -40,7 +40,7 @@ Verify that you have the latest version of Go installed:
 
 ```sh
 $ go version
-go version go1.13 darwin/amd64
+go version go1.13.1 darwin/amd64
```

Comment:
```suggestion
go version go1.13.* darwin/amd64
```

since we support all versions of 1.13, just so the user knows they can ues any version?

Reply:
But it's the output of go version command. Let's maybe add a sentence like `Tendermint requires Go versions >= 1.12`

Reply:
`Verify that you have the latest version of Go installed` should be enough imo